com.unity.render-pipelines.core/Runtime/Textures/Texture2DAtlas.cs

```diff
@@ -375,13 +413,13 @@ public virtual void BlitTexture(CommandBuffer cmd, Vector4 scaleOffset, Texture
         /// <param name="overrideInstanceID">Override texture instance ID.</param>
         public virtual void BlitOctahedralTexture(CommandBuffer cmd, Vector4 scaleOffset, Texture texture, Vector4 sourceScaleOffset, bool blitMips = true, int overrideInstanceID = -1)
         {
-            // This atlas only support 2D texture so we only blit 2D textures
-            if (Is2D(texture))
-                BlitOctahedralTexture(cmd, scaleOffset, texture, sourceScaleOffset, blitMips);
+            // TODO: verify, what does pot2 padding for allocator do?
```

Comment:
Is this on your todo list for a follow-up PR? Just wondering if this TODO might end up in the codebase forever...

Reply:
Yeah, I can remove it. I need to read the allocator code in more detail at some point.